Diffusion/Pipeline/PipelineLoader.swift

```diff
@@ -64,11 +62,21 @@ class PipelineLoader {
 }
 
 extension PipelineLoader {
+    // Note: this function is currently unused. Marked for deletion. -- dolmere
```

Comment:
```suggestion
    // Unused. Kept for debugging purposes.
```

I sometimes use it when working on the download code; I just invoke it from `.onAppear` or whatever. But it's ok to remove it if you prefer so.